rename UserLeap destination to Sprig Cloud, edits on Sprig (Actions) (#2359)
segmentio/segment-docs: src/connections/destinations/catalog/actions-sprig-web/index.md

- [Track Event](#track-event)
- [Update User ID](#update-user-id)

<!-- If applicable, add information regarding the migration from a classic destination to an Actions-based version below -->

## Migration from the classic Sprig destination

> To prevent duplicate events being created in Sprig, ensure that for each Segment source, this destination and the Sprig Cloud destination are not both enabled at the same time.

Reviewer: ```suggestion
To prevent the creation of duplicate events in Sprig, ensure that this destination and the Sprig Cloud destination aren't enabled at the same time for each Segment source.
```

> To prevent duplicate events being created in Sprig, ensure that for each Segment source, this destination and the Sprig Cloud destination are not both enabled at the same time.

Reviewer: ```suggestion
To prevent the creation of duplicate events in Sprig, ensure that this destination and the Sprig Cloud destination aren't enabled at the same time for each Segment source.
```
